test/integration/060_persist_docs_tests/models/quote_model.sql

```diff
@@ -0,0 +1 @@
+select 1 as {{ adapter.quote("test2id") }}
```

Comment:
Unfortunately, I don't think this is a very useful test for postgres/redshift. I know columns that start with numbers aren't allowed in bigquery, but this column doesn't require quoting on postgres/redshift and would work fine regardless, so that's not much of a test. Snowflake will have case-sensitivity issues so it's probably still useful there.

I think you'll have to turn this into its own test case. I have done a bit of reading and I actually think that it's fine to exclude Bigquery from this test entirely: since the only things allowed in column names don't require quoting, and unquoted column names are case-preserving, quoting columns on Bigquery seems to be totally meaningless (identifiers are less restrictive, so quoting matters there: hyphens come to mind).